### PR TITLE
[mangrove-station-qv-scaled-to-mgv] Add Mangrove Station GroupOS voting strategy

### PR DESCRIPTION
**Pull request by espendk**
_This PR depends on https://github.com/snapshot-labs/snapshot-strategies/pull/1316 and https://github.com/snapshot-labs/snapshot-strategies/pull/1315._

This PR adds the `mangrove-station-qv-scaled-to-mgv` strategy. The strategy allows active members of one of [Mangrove DAO's](https://www.mangrove.exchange/) governance groups (Builders Group and Pods Group) to vote in the Mangrove multistakeholder governance.

Quadratic voting is applied to each member's score and then the voting weight is scaled such that the total voting weight of the group is equal to the circulating supply of MGV tokens. This ensures that the three stakeholder groups have equal voting weights.

## Notes
Mangrove DAO governance groups are [Station](https://www.station.express/) [GroupOS](https://groupos.xyz/) groups on-chain, and active members have a ERC-1155 badge.

The circulating supply of MGV tokens is the total number of allocated, claimable tokens.

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @espendk One PR should contain changes of one strategy, else it will be hard to test or review

**Comment by espendk** [comment]
> Hi @espendk One PR should contain changes of one strategy, else it will hard to test or review 

Ok. But I'm unsure how to do that when submitting multiple, interdependent strategies for review as is the case here?

File: src/strategies/mangrove-station-qv-scaled-to-mgv/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  // Get the circulating supply of the vesting token
+  const allVestings = await getAllVestings(
+    network,
+    provider,
+    snapshot,
+    options.dssVestAddress,
+    options.dssVestDecimals
+  );
```
Comment (ChaituVR) [inline src/strategies/mangrove-station-qv-scaled-to-mgv/index.ts:75]: This is crossing the limit of 5 requests per strategy 🙈 you see some other way to do this???

https://github.com/snapshot-labs/snapshot-strategies#code

Comment (ChaituVR) [reply src/strategies/mangrove-station-qv-scaled-to-mgv/index.ts:75]: Everything else looks good

Comment (espendk) [reply src/strategies/mangrove-station-qv-scaled-to-mgv/index.ts:75]: Uh oh. I don't see any other way to do this w/o resorting to some off-chain indexing :-/

Comment (ChaituVR) [reply src/strategies/mangrove-station-qv-scaled-to-mgv/index.ts:75]: I think this request is fine as it is very light

**Pull request merged**